### ISSUE TITLE
[Grails] Hot key support for groovy file /gsp file/html file

### ISSUE THREAD

**Issue by feather812002** [prominic/Moonshine-IDE#599]
I can success import a grails priject into Moonshine-IDE.
But I found it is difficult when I try coding with Moonshine-IDE.
I no found any hot key work with Moonshine-IDE.
For example ,when I want comment one line code . 
I hit the "Command+ /" in the keybord , but nothing happen .
This just a simple hot key ,it work with any IDE .
I don't know why it not support in Moonshine-IDE.

Other important thing is if we want the Moonshine-IDE work in real project , we must need 
it prompt Variable name, method name ,etc. and And auto-complete them.
Any programer don't want  remember all  variable name, method name, and so on.

So in short ,on my side ,until now ,I only success import the grails project .
But can't work with moon ide any more.

I don't found it have  any different if I open a grails project with notespad program .

**Comment by rat-moonshine** [comment]
Hi, thanks for reporting. 

We are aware that many shortcut-keys that are not yet present in Moonshine, but we're working on integrating most basic needs to the editor/IDE. Some issues already raised on needful shortcut-keys which you can find here (in Issue list). We also have a plan to expand the possibility of customizing shortcut-keys in future (#588 ). For now, you can find the most of the shortcut-keys are defined in top-menu options.

On auto-completion/language-server to Grails project, I believe we already have some functionalities in added - @JoelProminic can confirm this better. 

However, when you import a Grails project, and before starting a language-server to Grails project, Moonshine needs to update the project's class-paths. You can see some console-output like this (following) when after importing a Grails project to Moonshine, AND before starting the language-server:

```
: |Application created at C:\Users\devsena\Desktop\NewGrailsProject
: > Task :eclipseClasspath
: > Task :eclipseJdt
: > Task :eclipseProject
: > Task :eclipseWtpComponent
: > Task :eclipseWtpFacet
: > Task :eclipseWtp
: > Task :eclipse
: Deprecated Gradle features were used in this build, making it incompatible with Gradle 6.0.
: Use '--warning-mode all' to show the individual deprecation warnings.
: See https://docs.gradle.org/5.4.1/userguide/command_line_interface.html#sec:command_line_warnings
: 
: BUILD SUCCESSFUL in 2s
: 5 actionable tasks: 5 executed
```

If the above process failed you for some reason, then I think then it is possible that the language-server did not start for you properly (you can check the console output after Grails project import).

**Comment by feather812002** [comment]
okay ,it look like I am not starting the  language-server  correct.
But ,if the moon-ide must need the starting a language-server to Grails project, the correct way should report a alert message tell user how to starting the language-server.

I am a newbie for Moonide.
I think any user work with moon-ide first time  should get the correct alert message, 
It can guidness me how to starting the  language-server 
make sure let me work in the correct way .


I am sure  @JoelProminic can work fine with this  
But if the moon-ide want other new user work with it , it must more friendly for new user , rather than just work with developer of moon-ide.

**Comment by JoelProminic** [comment]
We also have specific issue for block commenting:  #575

For the language server problems, this sounds like he same behavior as #591.  This is stuck until we can provide Josh with a public Grails project that he can use to reproduce the error.  Hopefully the phone interface app will be released soon.
